scripts/download.pl

```diff
@@ -274,17 +274,17 @@ sub cleanup
 	} elsif ($mirror =~ /^\@GNU\/(.+)$/) {
 		push @mirrors, "https://mirror.csclub.uwaterloo.ca/gnu/$1";
 		push @mirrors, "https://mirror.netcologne.de/gnu/$1";
-		push @mirrors, "http://ftp.kddilabs.jp/GNU/gnu/$1";
-		push @mirrors, "http://www.nic.funet.fi/pub/gnu/gnu/$1";
-		push @mirrors, "http://mirror.navercorp.com/gnu/$1";
+		push @mirrors, "https://ftp.kddilabs.jp/GNU/gnu/$1";
+		push @mirrors, "https://www.nic.funet.fi/pub/gnu/gnu/$1";
+		push @mirrors, "https://mirror.navercorp.com/gnu/$1";
 		push @mirrors, "https://mirrors.rit.edu/gnu/$1";
 		push @mirrors, "https://ftp.gnu.org/gnu/$1";
 		push @mirrors, "https://mirrors.tuna.tsinghua.edu.cn/gnu/$1";
 		push @mirrors, "https://mirrors.ustc.edu.cn/gnu/$1";
 	} elsif ($mirror =~ /^\@SAVANNAH\/(.+)$/) {
 		push @mirrors, "https://mirror.netcologne.de/savannah/$1";
 		push @mirrors, "https://mirror.csclub.uwaterloo.ca/nongnu/$1";
-		push @mirrors, "http://ftp.acc.umu.se/mirror/gnu.org/savannah/$1";
+		push @mirrors, "https://ftp.acc.umu.se/mirror/gnu.org/savannah/$1";
```

Comment:
This mirror is listed 2 times

Reply:
duplicate removed.  (had been both http and ftp, thus the duplicate)